### PR TITLE
mobile repsonsiveness fix

### PR DESCRIPTION
**Pull request by rachelhox**
## Checklist
- [ ] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: src/components/navbar/components/mobile/index.tsx [hunk 1]
```diff
@@ -17,7 +17,9 @@ const MobileNav = (props: IMobileNav) => {
         color={color}
         displayBackground={displayBackground}
         className={classNames("icon")}
-      ></MobileNavBarIconCSS>
+      >
+        <div></div>
```
Comment (kwunyeung) [inline src/components/navbar/components/mobile/index.tsx:21]: What's the usage of this extra `<div>`?

Comment (rachelhox) [reply src/components/navbar/components/mobile/index.tsx:21]: `<div>/<div>` has a css style applied, see
https://github.com/forbole/official-website/blob/412f3bd9636f0b8a1c2628b6c7837b795285e4da/src/components/navbar/components/mobile/styles.tsx#L21-L31

Comment (rachelhox) [reply src/components/navbar/components/mobile/index.tsx:21]: if there's no `<div>/<div>`, the burger icon in mobile view would look like this
image

Comment (kwunyeung) [reply src/components/navbar/components/mobile/index.tsx:21]: I see

**Review by kwunyeung — APPROVED** [review]
ACK

**Pull request merged**